namer/comparison_results.py

```diff
@@ -167,8 +168,8 @@ def as_dict(self, config: NamerConfig):
             "name": self.name,
             "site": self.site.replace(" ", "") if self.site else None,
             "full_site": self.site,
-            "performers": " ".join(map(lambda p: p.name, filter(lambda p: p.role == "Female", self.performers))) if self.performers else None,
-            "all_performers": " ".join(map(lambda p: p.name, self.performers)) if self.performers else None,
+            "performers": ", ".join(map(lambda p: p.name, filter(lambda p: p.role == "Female", self.performers))) if self.performers else None,
```

Comment:
This will change behavior for anyone wanting no commas between performer names, we'll have to provide info on how to use replace to get rid of it in the config

Reply:
they can do `{performers:|replace(', ', ' ')}`, if someone want commas he can't do anything

Reply:
Agreed, just needs in the release notes.